src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf)  format("turetype");
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.input {
+  font-family: "Avenir", serif;
+  font-weight: 300;
+  font-style: normal;
+  box-sizing: border-box;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  padding: 25px 0 23px 62px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.input__small {
+  height: 42px;
+  padding: 12px 0 10px 33px;
+  font-size: 14px;
+  line-height: 19px;
+  /* font-weight: 400; */
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input__small::placeholder {
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+  height: 20px;
+}
+
+.input:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input__small:focus {
+  font-weight: 900;
+}
+
+.search-input {
+  position: relative;
+  display: block;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+  display: block;
+
```

Comment:
remove empty line please

Reply:
I tried many times but it was unsuccessful with test